docs/datasource_api_userguide.md

```diff
@@ -308,7 +300,9 @@ The following table shows the TiDB-specific options, which can be passed in thro
 
 ## TiDB Version and Configuration for Write
 
-TiDB's version must be 3.0 or later.
+TiDB's version must be 4.0 or later.
+
+**IMPORTANT: currently TiDB-4.0 is not released yet, but you can use code on tidb's master branch.**
```

Comment:
s/can use code on tidb's master branch./can use tidb's master branch. 

Additionally, any requirement of TiKV and pd?

Reply:
fixed.

requirement of TiKV and pd is that: compatible with tidb master branch